tests/schema/tests.py

```diff
@@ -2303,6 +2304,54 @@ class Meta:
         columns = self.column_classes(Author)
         self.assertEqual(columns["birth_year"][1].default, "1988")
 
+    @isolate_apps("schema")
+    def test_add_text_field_with_db_default(self):
+        class Author(Model):
+            description = TextField(db_default="(missing)")
```

Comment:
This was crashing on MySQL due to improper handling of literal on some _special_ fields.